ads/google/imaVideo.js

```diff
@@ -683,6 +699,13 @@ export function onContentEnded() {
   if (adsLoader) {
     adsLoader.contentComplete();
   }
+
+  // If all ads are not completed,
+  // onContentResume will show the bigPlayDiv
+  if (allAdsCompleted) {
+    setStyle(bigPlayDiv, 'display', 'table-cell');
```

Comment:
Why `table-cell`?

Reply:
Trying to stay consistent with other control element styling, and don't want to cause any weird CSS bugs by changing the way the elements are being laid out.

Definitely agree the styling on the controls should be improved though. Hopefully in a later refactor if I get an "Okay" on that.

What do you think? 😄

Reply:
SGTM, but, if possible, try to keep this value in a single source. Either by reading the calculated style or by setting it some other way by default.